test_unstructured/partition/pdf_image/test_chipper.py

```diff
@@ -1,13 +1,14 @@
 import pytest
 
 from unstructured.partition import pdf
+from unstructured.partition.utils.constants import PartitionStrategy
```

Comment:
do we want a changelog item to advertise that we got this class for the constants now?

Reply:
Sounds good. Thanks!